data/json/items/tool/misc.json

```diff
@@ -227,6 +227,24 @@
       "done_message": "You unroll the fur mat and lay it on the ground."
     }
   },
+  {
+    "id": "hose_garden",
+    "looks_like": "hose",
+    "type": "TOOL",
+    "name": { "str": "garden hose" },
+    "description": "This is a flexible garden hose.  If you cut it in smaller pieces, it could be used for crafting, or siphoning fuel from a vehicle.",
+    "weight": "1632 g",
+    "volume": "1500 ml",
+    "longest_side": "100 cm",
```

Comment:
I would argue that spooled garden hose would be less than 1m in diameter.
To not be anecdotical:
![hoses](https://user-images.githubusercontent.com/26584783/84480961-09f3bc00-ac9e-11ea-851f-1b559a7f1c28.jpg)
(Unless you think the person in this picture is 4+ metres tall)

Reply:
I agree it could be changed to 0.5m, I thought of it without the spool, just the hose. Will see what others think, but probably will change to 0.5 or even 0.3 in this case.

Reply:
Oh yeah, it's in my understanding that longest side is something that can't be reduced - like a rifle or a shovel can't be folded further to reduce length, but a hose can be spooled or just crumpled to a size smaller than 1m.

Reply:
Good point, in that case I shall reduce it to 30 cm. Looking at the photo, that guy is around 6 of the garden hoses (without the spool) tall. If we consider him 1.8m tall / 6 = 30 cm, just on point.